site/_includes/layouts/doc-post.njk

```diff
@@ -1,7 +1,7 @@
 {% extends "layouts/base.njk" %}
 
 {% block title_bar %}
-  {% include 'partials/breadcrumbs.njk' %}
+  {% include 'partials/breadcrumbs-post.njk' %}
```

Comment:
Initially I wasn't so sure about naming this "-post" because then it seems like it is just for posts, but we use it for docs as well.

We could keep the same breadcrumb include and make the share button configurable.

```njk
{% set sharingEnabled = true %}
{% include 'partials/breadcrumbs.njk' with context %}
```

By adding `with context` we give the partial access to variables in the parent scope.

Alternatively we could convert the breadcrumbs partial into a macro and pass it as an argument.

```njk
{% set options = { sharingEnabled: true } %}
{{ breadcrumbs(options) }}
```

Note that nunjucks has some issues with using set and object sub properties (https://github.com/mozilla/nunjucks/issues/313)

Reply:
I think "with context" works only with import, not include. with include the variable should just be available inside the template as-is.